Gemfile.lock

```diff
@@ -3,7 +3,7 @@ GEM
   specs:
     CFPropertyList (3.0.5)
       rexml
-    activesupport (6.1.4.3)
+    activesupport (6.1.6)
```

Comment:
Had several issues with docs generation using Jazzy. Had to remove `Gemfile.lock` and fully update dependencies.